templates/web-application/postcss.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('postcss-load-config').Config} */
```

Comment:
i would expect /** @type {import("postcss").Postcss} */

instead of the load-config import

Reply:
It doesn't work with `/** @type {import("postcss").Postcss} */`. This import is based on PostCSS documentation https://github.com/postcss/postcss#css-in-js

Reply:
i would remove the JSDoc typings then. It's not worth importing the package